website/src/components/Layout.tsx

```diff
@@ -41,10 +41,11 @@ export default class Layout extends React.PureComponent<LayoutProps> {
         const isHome = pathname === '/'
         const isBlog = pathname === '/blog'
         const isProductPage = pathname.startsWith('/product/')
+        const isCaseStudyPage = pathname.startsWith('/case-studies/') && pathname.split('/')[2] !== ''
```

Comment:
Super minor: could we just say `pathname !== '/case-studies'`?

Reply:
The way I have it now is a bit more fool proof since the trailing slash gets added to the URL automatically if you refresh the page after landing on it.

Reply:
Ah right, I forgot that Gatsby enforces that. Sounds good!